snuba/datasets/configuration/issues/entities/search_issues.yaml

```diff
@@ -0,0 +1,46 @@
+version: v1
+kind: entity
+name: search_issues
+
+schema:
+  [
+    { name: organization_id, type: UInt, args: { size: 64 } },
+    { name: project_id, type: UInt, args: { size: 64 } },
+    { name: group_id, type: UInt, args: { size: 64 } },
+    { name: search_title, type: String },
+    { name: fingerprint, type: Array, args: { inner_type: { type: String } } },
+    { name: occurrence_id, type: UUID },
+    { name: occurrence_type_id, type: UInt, args: { size: 8 } },
+    { name: detection_timestamp, type: DateTime },
+
+    # TODO: fill out more fields
+  ]
+
+readable_storage: search_issues
+writable_storage: search_issues
+query_processors:
+  - processor: OrganizationRateLimiterProcessor
+    args:
+      org_column: organization_id
+  - processor: ReferrerRateLimiterProcessor
+  - processor: ProjectReferrerRateLimiter
+    args:
+      project_column: project_id
+  - processor: ProjectRateLimiterProcessor
+    args:
+      project_column: project_id
+  - processor: ResourceQuotaProcessor
+    args:
+      project_field: project_id
+  - processor: TimeSeriesProcessor
```

Comment:
I would suggest that you don't need this processor. This allows some syntactic sugar, converting a "magic" column into a bucketed time (for example, magic `time` column becomes `toStartOfHour(timestamp)`). This logic isn't really useful for new Datasets since you can do that with SnQL now, and it will just cause headaches when trying to debug queries.